removeDuplicatesFromSortedListII.py

```diff
@@ -10,10 +10,12 @@ def deleteDuplicates(self, head):
         :type head: ListNode
         :rtype: ListNode
         """
-        dummy = ListNode(2 ** 31 + 1)
+        if not head:
+            return head
+        dummy = ListNode(head.val - 1)
         preNode = dummy
         
-        preVal = 2 ** 31 + 1
+        preVal = head.val - 1
```

Comment:
Define `preVal` above the definition of `dummy` and `preNode`.
Then you can do `dummy = preNode = ListNode(preVal)`